apis/org_handler_test.go

```diff
@@ -90,56 +308,56 @@ func testListingOrgs(t *testing.T, when spec.G, it spec.S) {
 
 		it("renders the orgs response", func() {
 			expectedBody := fmt.Sprintf(`
-                {
-                   "pagination": {
-                      "total_results": 2,
-                      "total_pages": 1,
-                      "first": {
-                         "href": "%[1]s/v3/organizations?page=1"
-                      },
-                      "last": {
-                         "href": "%[1]s/v3/organizations?page=1"
-                      },
-                      "next": null,
-                      "previous": null
-                   },
-                   "resources": [
-                        {
-                            "guid": "a-l-i-c-e",
-                            "name": "alice",
-                            "created_at": "2021-09-17T15:23:10Z",
-                            "updated_at": "2021-09-17T15:23:10Z",
-                            "suspended": false,
-                            "metadata": {
-                              "labels": {},
-                              "annotations": {}
-                            },
-                            "relationships": {},
-                            "links": {
-                                "self": {
-                                    "href": "%[1]s/v3/organizations/a-l-i-c-e"
-                                }
-                            }
-                        },
-                        {
-                            "guid": "b-o-b",
-                            "name": "bob",
-                            "created_at": "2021-09-17T15:23:10Z",
-                            "updated_at": "2021-09-17T15:23:10Z",
-                            "suspended": false,
-                            "metadata": {
-                              "labels": {},
-                              "annotations": {}
-                            },
-                            "relationships": {},
-                            "links": {
-                                "self": {
-                                    "href": "%[1]s/v3/organizations/b-o-b"
-                                }
-                            }
-                        }
-                    ]
-                }`, rootURL)
+{
```

Comment:
It looks like this line was incorrectly indented by accident.

Reply:
Indentation was a bit dodgy in that file. Fixed now (I think)